libs/platform/router/src/services/default-router.service.ts

```diff
@@ -21,6 +22,16 @@ import {
 const CURRENT_PAGE = 'currentPage';
 const PREVIOUS_PAGE = 'previousPage';
 
+const isPatternConfig = (route: RouteConfig): route is URLPatternRouteConfig =>
+  (route as URLPatternRouteConfig).pattern !== undefined;
+
+const getPattern = (route: RouteConfig) => {
```

Comment:
Just add `export` in `lit-router` don't do copy paste

Reply:
It is not just a copy-paste, the implementation in service was simplified. We discussed that with Stan, and decided not to touch the router (manage exports, change descriptors) due to router is temporary solution and can be changed or even dropped in the future